apidoc/Titanium/UI/ListView.yml

```diff
@@ -1058,6 +1058,52 @@ properties:
     osver: {ios: {min: "7.0"}}
     platforms: [iphone, ipad]
 
+  - name: resultsBackgroundColor
+    summary: The background color of the search results (iOS-only).
+    description: |
+        For information about color values, see the "Colors" section of <Titanium.UI>. 
+        Note: A transparent background-color is not officially supported by Apple to
+        prevent that the list of results overlaps with the list view below it.  
+    type: String
+    default: undefined (behaves as white) 
+    since: 6.1.0
+    platforms: [iphone, ipad]
+    availability: creation
+    removed: 
+    since: "6.2.0"
+    notes: |
+        The background color of the search table-view is determined by it's table-view, so there is no additional property required.
+
```

Comment:
This is not the way the docs are written. Please check how other docs handle removal, also remember to break long lines into separate ones.